### PR TITLE
refactor: flake8 for io

### PR DESCRIPTION
**Pull request by synchon**
* [ ] fix #(issue number)
* [x] description of feature/fix
* [x] tests added/passed
* [ ] add an entry to the [next release](../release-notes/next-release.md)

This PR resolves `flake8` issues for `cobra.io` and `test_io` and upgrades code style wherever required.

### CODE REVIEW

File: src/cobra/io/web/cobrapy_repository.py [hunk 1]
```diff
@@ -3,8 +3,6 @@
 
 import importlib_resources
 
-import cobra.data
```
Comment (Midnighter) [inline src/cobra/io/web/cobrapy_repository.py]: Please leave this import but remove the quotes from the line 52.

File: src/cobra/io/web/load.py [hunk 2]
```diff
@@ -25,12 +25,17 @@
 configuration = Configuration()
 
 
+cobrapy_repository = Cobrapy()
+bigg_models_repository = BiGGModels()
+bio_models_repository = BioModels()
```
Comment (Midnighter) [inline src/cobra/io/web/load.py]:
```suggestion
DEFAULT_REPOSITORIES = (
    Cobrapy(),
    BiGGModels(),
    BioModels(),
)
```

File: src/cobra/io/web/load.py [hunk 3]
```diff
@@ -25,12 +25,17 @@
...
+
 def load_model(
     model_id: str,
     repositories: Iterable[AbstractModelRepository] = (
-        Cobrapy(),
-        BiGGModels(),
-        BioModels(),
+        cobrapy_repository,
+        bigg_models_repository,
+        bio_models_repository,
```
Comment (Midnighter) [inline src/cobra/io/web/load.py]: Change to fit above suggestion.

File: tests/test_io/test_web/test_load.py [hunk 4]
```diff
@@ -1,68 +1,116 @@
...
+
+    from cobra import Model
+
+
 @pytest.fixture(scope="module")
-def mini_sbml(data_directory):
+def mini_sbml(data_directory: Path) -> bytes:
     """Provide a gzip-compressed SBML document."""
-    with (pathlib.Path(data_directory) / "mini_cobra.xml").open(mode="rb") as handle:
+    with open(data_directory / "mini_cobra.xml", "rb") as handle:
```
Comment (Midnighter) [inline tests/test_io/test_web/test_load.py]:
```suggestion
    with (data_directory / "mini_cobra.xml").open(mode="rb") as handle:
```

**Review by Midnighter — CHANGES_REQUESTED** [review]
Nice, thanks for the fixes. Just a few smaller comments.

File: tests/test_io/test_web/test_load.py (hunk 4 above)
Comment (synchon) [reply tests/test_io/test_web/test_load.py]: Any specific reason for using `Path.open()`?

Comment (Midnighter) [reply tests/test_io/test_web/test_load.py]: No other reason than that `data_directory` is a `Path` and thus the method exists 😛

Comment (synchon) [reply tests/test_io/test_web/test_load.py]: Okay! 😆

**Comment by synchon** [comment]
Should be good for merging.

**Review by Midnighter — APPROVED** [review]

**Pull request merged**